pub/lib/dependabot/pub/helpers.rb

```diff
@@ -138,9 +144,12 @@ def ensure_right_flutter_release
           end
 
           parsed = JSON.parse(stdout)
+          flutter_version = parsed["frameworkVersion"]
+          dart_version = parsed["dartSdkVersion"].split.first
```

Comment:
```suggestion
          dart_version = parsed["dartSdkVersion"]&.split&.first
```

Should we add this to safe-guard against `parsed["dartSdkVersion"]` being nil? This would now raise an error if it's nil.

Reply:
We should always have a dart version. I made the check such that the error message will become nicer.